### DISCUSSION TITLE
如何让回答的结果停留时间更长些？

### DISCUSSION THREAD

**Discussion by xxhk-org** [Yue-Yang/ChatGPT-Siri#6] [Ideas]
现在答案出现后一两秒就自动关闭了，甚至来不及阅读。

**Comment by Yue-Yang** [comment]
这个需要修改快捷指令，如果你懂的话，可以自己试试。你可以手动点击运行，就是对话框输入模式可以复制答案了

后面我会更新一个版本，支持把对话写入到 备忘录保存起来

**Reply by SunDD4** [reply]
我在测试中也发现了两个问题：
一，设置里，如果SIRI的回答是“首选静音”，API获取来的答案停留时间很短，来不及看。。就跳到下一个语音指令。 
二，Siri设置选“自动”，且手机调静音的话则情况同上。如果手机未静音，让Siri语音回答（一是怕吵到别人，二如果Siri 未语音答完，按语音按钮中断它并输入新的问题时，Siri 会自动调取苹果自身Siri的工作模式，且自行中断ChatGPT的上下文关联回答。。

建议：如果在Siri“静音模式”下，让答案悬停。重新点击语音输入指令还能衔接上下文。
